zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -95,6 +95,7 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
     $scope.chart = {};
     $scope.colWidthOption = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
     $scope.paragraphFocused = false;
+    $scope.editor.setReadOnly($scope.isRunning());
```

Comment:
Sorry, my bad for pointing out that place, editor is not loaded yet at that point, so we have an error in the console. The right place should be around Line 542